### DISCUSSION TITLE
R-Ladies São Paulo - Raspando a página do Studio Ghibli, na Wikipédia

### DISCUSSION THREAD

**Discussion by giscus** [R-Ladies-Sao-Paulo/blog#35] [General]
# R-Ladies São Paulo - Raspando a página do Studio Ghibli, na Wikipédia

https://rladies-sp.org/posts/2023-10-ws-ghibli/

**Comment by IanniMuliterno** [comment]
que post lindo S2, gosto muito dos filmes do estúdio Ghibli
